java/pom.xml

```diff
@@ -145,10 +145,11 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <!-- Depend on `org.apache.hadoop.util.ShutdownHookManager` in `MemoryCleaner` -->
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-common</artifactId>
             <version>3.2.3</version>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
I prefer `provided` to `compile` according to your comment https://github.com/rapidsai/cudf/pull/11161#issuecomment-1169462324

Reply:
Agree about 'provided'  to avoid pulling in Hadoop in the shade plug-in

Reply:
`databricks` class path may not contain org.apache.hadoop.util.ShutdownHookManager. @gerashegalov

Reply:
may not? we should see this at the compile time during Db build. Please add `[databricks]` to the PR title to double-check.

Reply:
Done.